plugins/build.sbt

```diff
@@ -6,7 +6,7 @@ def finagle(mod: String) =
   "com.twitter" %% s"finagle-$mod" % "6.44.0"
 
 def linkerd(mod: String) =
-  "io.buoyant" %% s"linkerd-$mod" % "1.0.2"
+  "io.buoyant" %% s"linkerd-$mod" % "1.1.1"
```

Comment:
We also upgraded finagle / util as part of this release, so you should update those versions as well. Actually, I'm going to do the same for the linkerd-zipkin repo, if you want to hold off until I put that up for review. Should be pretty straightforward.